src/test/java/reposense/util/FileUtilTest.java

```diff
@@ -38,14 +38,14 @@ public void zip_invalidLocation_fail() {
 
     @Test
     public void zip_validFileType_success() throws IOException {
-        FileUtil.zip(OUTPUT_DIRECTORY_ABSOLUTE, ".csv", ".zip");
+        FileUtil.zip(OUTPUT_DIRECTORY_ABSOLUTE, ".csv");
         Assert.assertTrue(Files.exists(ARCHIVE_ZIP_PATH));
         Assert.assertTrue(Files.size(ARCHIVE_ZIP_PATH) > 0);
     }
 
     @Test
     public void zip_invalidFileType_fail() {
-        FileUtil.zip(REPORT_DIRECTORY_ABSOLUTE, ".jsonnn");
+        FileUtil.zip(REPORT_DIRECTORY_ABSOLUTE, ".pdf");
         Assert.assertFalse(Files.exists(ARCHIVE_ZIP_PATH));
```

Comment:
Add a comment that the 'empty' zip will return 'false' for the exist test, even though it gets created.